xarray/convert.py

```diff
@@ -49,3 +62,142 @@ def set_cdms2_attrs(var, attrs):
     cdms2_var = cdms2.createVariable(var.values, axes=axes, id=dataarray.name)
     set_cdms2_attrs(cdms2_var, var.attrs)
     return cdms2_var
+
+
+# TODO: Add converting bounds from xarray to Iris and back
+def to_iris(dataarray):
+    """Convert a DataArray into a Iris Cube
+    """
+    # Iris not a hard dependency
+    import iris
+    # iris.unit is deprecated in Iris v1.9
+    import cf_units
+
+    def check_attrs(attrs, keys):
+        return dict((k, v) for k, v in attrs.items() if k in keys)
+
+    def get_args(attrs):
+        _args = {'attributes': filter_attrs(attrs, iris_forbidden_keys)}
+        _args.update(check_attrs(attrs, ('standard_name', 'long_name',)))
+        _unit_args = check_attrs(coord.attrs, ('calendar',))
+        if 'units' in attrs:
+            _args['units'] = cf_units.Unit(attrs['units'], **_unit_args)
+        return _args
+
+    def get_cell_methods(cell_methods_str):
+        """Converts string to iris cell method objects"""
+        cell_methods = []
+        _cell_method_words = [w.strip() for w in cell_methods_str.split(':')]
+        cm = {'coords': [], 'method': '', 'interval': [], 'comment': []}
+        skip = False
+        for i, word in enumerate(_cell_method_words):
+            # If this value is a comment or an interval don't read
+            if skip:
+                skip = False
+                continue
+            # If this word is an axis
+            if word not in cell_methods_strings | set(['interval', 'comment']):
```

Comment:
Yes there may, but it is actually a little complicated to parse these strings. The format is:

"coord1: [coord2:] method: [comment: a comment] [interval: a interval]"

I.e. here are some examples:

```python
>>> get_cell_methods('dim1: dim2: mean: comment: A comment: interval: A interval')
[CellMethod(method='mean', coord_names=('dim1', 'dim2'), intervals=('A interval',), comments=('A comment',))]
>>> get_cell_methods('dim1: dim2: mean: dim2: sum:')
[CellMethod(method='mean', coord_names=('dim1', 'dim2'), intervals=(), comments=()), CellMethod(method='sum', coord_names=('dim2',), intervals=(), comments=())]
>>> get_cell_methods('dim1: mean: dim2: dim3: sum: comment: A comment')
[CellMethod(method='mean', coord_names=('dim1',),  intervals=(), comments=()), CellMethod(method='sum', coord_names=('dim2', 'dim3'), intervals=(), comments=('A comment',))]
```

So that line is looking to see if the word is a method or "comment" or "interval", if not it's a coord and so checks to see if it's a new cell method or another coord for the current one.